src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/ChunkOutputWidget.ui.xml

```diff
@@ -117,7 +117,7 @@
    
    .outer
    {
-      padding-bottom: 10px;
+      padding-bottom: 7px;
```

Comment:
Note: without this change, the new rounding behavior in the line widget row height computation causes the lower padding for the widget to become much larger than expected. 7px is still just enough breathing room (IMHO).